openquake/hazardlib/gsim/bradley_2013.py

```diff
@@ -46,6 +46,16 @@
      (172.6220, -43.5233)])
 
 
+def _get_basin_term(C, z1pt0):
+    """
+    Get the basin term
+    """
```

Comment:
As a general convention in the engine we do not add docstrings to private functions (i.e. starting with `_`)